EstateReporting.IntegrationTests/Common/Setup.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace EstateReporting.IntegrationTests.Common
+{
+    using System;
+    using Ductus.FluentDocker.Services;
+    using Ductus.FluentDocker.Services.Extensions;
+    using global::Shared.Logger;
+    using NLog;
+    using Shouldly;
+    using TechTalk.SpecFlow;
+
+    [Binding]
+    public class Setup
+    {
+        public static IContainerService DatabaseServerContainer;
+        private static String DbConnectionStringWithNoDatabase;
+        public static INetworkService DatabaseServerNetwork;
+
+        public static String SqlServerContainerName = "shareddatabasesqlserver";
```

Comment:
Codacy found an issue: [Change the visibility of 'SqlServerContainerName' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/stuart_ferguson1/EstateReporting/pullRequest?prid=6339123)